VERSION.txt

```diff
@@ -1 +1 @@
-1.22.1-dev
+1.22.1
```

Comment:
i think we should skip straight to 1.22.2 here

Reply:
Yes +1, can you add a commit to do this? We will need a new tag.

Reply:
yep, on it...